client/src/main/kotlin/no/nav/su/se/bakover/client/dokarkiv/JournalpostRequest.kt

```diff
@@ -147,5 +174,6 @@ enum class JournalPostType(val type: String) {
 
 enum class DokumentKategori(val type: String) {
     SOK("SOK"),
-    VB("VB")
+    VB("VB"),
```

Comment:
Står VB og IB for noe :grimacing: ?

Reply:
Vedtaksbrev, og infobrev. Er kodene som må brukes for journalposten. Jeg hadde infobrev som type på infobrev, med det gav meg en del issues, husker ikke om vet var tester eller ut i test. Kan kanskje legge på kommentar på hva det er.

Reply:
Jeg antar at DokumentKategori serialiseres direkte? Da vil den plukke enumen og ikke `type`. Da må du i så fall legge på en Jackson-annotasjon: `enum class DokumentKategori(@JsonValue val type: String)`

Reply:
👍